my-app/src/components/navbar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import style from "./navbar.module.css";
+import Link from "next/link";
+
+export default function navbar() {
+    return(

```

Comment:
Navbar component looks great- website is easy to navigate. Nice usage of the <Link> tags!